### DISCUSSION TITLE
Woocommerce Use Case

### DISCUSSION THREAD

**Discussion by basquith16** [dalenguyen/firebase-wordpress-plugin#153] [Ideas]
Hi Dale! I have a question about the Woocommerce extension. My current database for user licenses is set up as follows. When a user purchases a license for an event, it is attached to the user as an object (map) in an array

Can the extension add data in this manner? Or is the only current option to add it as a 'wooOrder' collection? The ability to customize where the data is added would be a game changer for me.

**Comment by dalenguyen** [comment]
Hi @basquith16, the order will be pushed to `wooOrders` collection (you can change the name though). Then you can add a firebase functions that listens to `wooOrders` document creation in order to map data to your `users` collection. The `wooOrders` has `userId` field that will help to map to the right user document.

**Reply by basquith16** [reply]
You've just made me a very happy man! I
